test/io.jl

```diff
@@ -516,4 +516,15 @@ module TestIO
     show(io, "text/html", df)
     str = takebuf_string(io)
     @test str == "<table class=\"data-frame\"><tr><th></th><th>Fish</th><th>Mass</th></tr><tr><th>1</th><td>Suzy</td><td>1.5</td></tr><tr><th>2</th><td>Amir</td><td>#NULL</td></tr></table>"
+
+    # test limit attribute of IOContext is used
+    df = DataFrame(a=collect(1:1000))
+    ioc = IOContext(IOBuffer(), displaysize=(10, 10), limit=false)
+    show(ioc, MIME{Symbol("text/html")}(), df)
+    @test ioc.io |> takebuf_string |> length > 10000
```

Comment:
Use standard syntax for nested function calls, as it's used elsewhere in the code. Also, maybe you could use `countlines` for a more precise check?